controllers/image_generator.go

```diff
@@ -98,6 +99,11 @@ func ImageCreate(c client.Client) (bool, ctrl.Result) {
 		}
 	}
 
+	if ig.provider == unsupportedCloudProvider {
+		igLogger.Info("unsupported cloud provider, skipping image creation")
+		return true, ctrl.Result{Requeue: true}
```

Comment:
It seems that we shouldn't requeue in this case.

Reply:
it won't, returning true means it will ignore the returning value, I can't return nil here, how about `ctrl.Result{}`?